docs/csharp/language-reference/keywords/foreach-in.md

```diff
@@ -31,6 +31,10 @@ The next example uses the `foreach` statement with an instance of the <xref:Syst
 
 [!code-csharp-interactive[span example](~/samples/snippets/csharp/keywords/IterationKeywordsExamples.cs#2)]
 
+Beginning with C# 7.3, when the collection type supports `ref` access to its elements, you can declare the iteration variable with the `ref` or `ref readonly` modifier. The following example uses a `ref` iteration variable to set the value of each item in a stackalloc array. The `ref readonly` version iterates the collection to print all the values. The `readonly` declaration uses an implicit local variable declaration. Implicit variable declarations can be used with either `ref` or `ref readonly` declarations, as can explicitly typed variable declarations.
```

Comment:
@BillWagner 
> ...when the collection type supports `ref` access to its elements...

Should that be clarified? How do I know when the collection type supports `ref` access to the elements? Is the only sufficient condition is that the `Current` property returns `ref T` (or it's just one of several)? If yes, is it fine to rephrase the following way:
>Beginning with C# 7.3, when the `Current` property returns `ref T` where `T` is the type of the collection element, you can declare the iteration variable with the `ref` or `ref readonly` modifier.

I don't like mentioning the `Current` property here. However, that is introduced anyway at the start of the topic.

Reply:
@pkulikov Your proposed change is great. Would you like to submit a PR for this?

Reply:
@BillWagner I'll create a PR with the change today.